Proposal-eng.md

```diff
@@ -0,0 +1,64 @@
+# Proposal - UIC HCC Computer Society
+
+Dear Sir / Madam:
+
+
+
+We are computer technology enthusiasts in UIC from different majors and different grades, who founded UIC HCC Computer Society (Unofficial) in October, 2016, in hope of spreading the knowledge of computer science and technology, conveying the sprit of open source, and sharing as well as focusing on the frontier development of computer science. Today, we wish to establish UIC HCC Computer Society officially under Innovation Center at the moment when UIC New Campus has moved smoothly and Innovation Center will be established soon. We hope to make contributions to students and staffs in UIC as well as the Industry-University Integration plan of UIC with the support of this platform.
+
+This proposal will introduce overview, vision, mission, task and expected achievement of UIC HCC Computer Society (hereafter this text will be abbreviated as Society). In addition, we will list our expectations to Innovation Center and UIC.
```

Comment:
1. the overview
2. missions, tasks, expected achievements
3. (hereinafter abbreviated as Society)
4. Simultaneously, we will also list our expectations to IC and UIC

Reply:
1. Will fix.
2. Will fix.
3. Good, will fix.
4. Will fix.